docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -252,10 +252,11 @@ see its help page:
 .. _Wheels: python-wheels
 .. _Linux packages: linux-packages
 .. _Create the queue repository: https://help.github.com/articles/creating-a-new-repository
```

Comment:
Might be updated to https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository ?

Reply:
Thanks @pitrou, I'll update this link